styles/index.scss

```diff
@@ -15,13 +15,29 @@ $heading-font: 'Nunito', 'Avenir', sans-serif;
   width: 100%;
 }
 
-.section{
+.top-photo {
+  width: 575px;
+  height: 530px;
```

Comment:
by specifying both width and height in CSS the image gets stretched to that size. usually you want to avoid doing this